startup/gui/outputs.py

```diff
@@ -185,45 +185,209 @@
+			{
+				"scalarformat" : "half",
+				"sortkey" : 1,
+				"filter" : "cryptomatteheader",
+			}
+		)
+	)
+
+	GafferScene.Outputs.registerOutput(
+		"Batch/3Delight/Crypto/Geometry_Cryptomatte_Layer0",
+		IECoreScene.Output(
+			"${project:rootDirectory}/renders/${script:name}/${renderPass}/id.geometry/id.geometry.####.exr",
+			"exr",
+			"quad builtin:id.geometry",
+			{
+				"scalarformat" : "float",
+				"sortkey" : 2,
+				"filter" : "cryptomattelayer0",
+				"customdrivername" : "Batch/3Delight/Crypto/Geometry_Cryptomatte_Header",
+			}
+		)
+	)
+
+	GafferScene.Outputs.registerOutput(
+		"Batch/3Delight/Crypto/Geometry_Cryptomatte_Layer2",
+		IECoreScene.Output(
+			"${project:rootDirectory}/renders/${script:name}/${renderPass}/id.geometry/id.geometry.####.exr",
+			"exr",
+			"quad builtin:id.geometry",
+			{
+				"scalarformat" : "float",
+				"sortkey" : 3,
+				"filter" : "cryptomattelayer2",
+				"customdrivername" : "Batch/3Delight/Crypto/Geometry_Cryptomatte_Header",
+			}
+		)
+	)
+
+	GafferScene.Outputs.registerOutput(
+		"Batch/3Delight/Crypto/Scene_Path_Cryptomatte_Header",
+		IECoreScene.Output(
+			"${project:rootDirectory}/renders/${script:name}/${renderPass}/id.scenepath/id.scenepath.####.exr",
+			"exr",
+			"color builtin:id.scenepath",
+			{
+				"scalarformat" : "half",
+				"sortkey" : 1,
+				"filter" : "cryptomatteheader",
+			}
+		)
+	)
+
+	GafferScene.Outputs.registerOutput(
+		"Batch/3Delight/Crypto/Scene_Path_Cryptomatte_Layer0",
+		IECoreScene.Output(
+			"${project:rootDirectory}/renders/${script:name}/${renderPass}/id.scenepath/id.scenepath.####.exr",
+			"exr",
+			"quad builtin:id.scenepath",
+			{
+				"scalarformat" : "float",
+				"sortkey" : 2,
+				"filter" : "cryptomattelayer0",
+				"customdrivername" : "Batch/3Delight/Crypto/Scene_Path_Cryptomatte_Header",
+			}
+		)
+	)
+
+	GafferScene.Outputs.registerOutput(
+		"Batch/3Delight/Crypto/Scene_Path_Cryptomatte_Layer2",
+		IECoreScene.Output(
+			"${project:rootDirectory}/renders/${script:name}/${renderPass}/id.scenepath/id.scenepath.####.exr",
+			"exr",
+			"quad builtin:id.scenepath",
+			{
+				"scalarformat" : "float",
+				"sortkey" : 3,
+				"filter" : "cryptomattelayer2",
+				"customdrivername" : "Batch/3Delight/Crypto/Scene_Path_Cryptomatte_Header",
+			}
+		)
+	)
+
 	for name, displayName, source, dataType in [
 		( "Ci", "Ci", "shader", "color" ),
-		( "Ci.direct", "Ci (direct)", "shader", "color" ),
-		( "Ci.indirect", "Ci (indirect)", "shader", "color" ),
+		( "Ci.direct", "Ci_(direct)", "shader", "color" ),
+		( "Ci.indirect", "Ci_(indirect)", "shader", "color" ),
 		( "diffuse", "Diffuse", "shader", "color" ),
-		( "diffuse.direct", "Diffuse (direct)", "shader", "color" ),
-		( "diffuse.indirect", "Diffuse (indirect)", "shader", "color" ),
-		( "hair", "Hair and Fur", "shader", "color" ),
-		( "subsurface", "Subsurface Scattering", "shader", "color" ),
+		( "diffuse.direct", "Diffuse_(direct)", "shader", "color" ),
```

Comment:
What's the reason for the " " -> "_" changes here? Is it to align with the other outputs, or is there a technical reason as well?

Reply:
The `customdrivername` implementation didn't work with spaces in the output name, so I had to replace them with `_`. With multipart output getting changed from using `customdrivername` to using the file name, there is no need for this change any more.